.eslintrc.json

```diff
@@ -12,6 +12,7 @@
         "curly": 1,
         "eol-last": 1,
         "eqeqeq": 1,
+        "getter-return": 1,
```

Comment:
In dev mode, only display warnings (don't fail).  Yes, it's a nice rule to have, but the places it says are errors are acceptable syntax.